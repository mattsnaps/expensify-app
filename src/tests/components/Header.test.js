import React from "react";
import { shallow } from 'enzyme';
import Header from "../../components/Header";

test('Render Header Success', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.find('h1').length).toBe(1);

    // const renderer = new ShallowRenderer();
    // renderer.render(<Header />);
    // expect(renderer.getRenderOutput()).toMatchSnapshot();
});
