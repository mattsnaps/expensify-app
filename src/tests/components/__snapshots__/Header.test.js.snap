// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Header Success 1`] = `
<header>
  <h1>
    Expensify App
  </h1>
  <NavLink
    className={[Function]}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    className={[Function]}
    to="/create"
  >
    Create Expense
  </NavLink>
</header>
`;
